### ISSUE TITLE
Bintray Shutdown Library dependacies

### ISSUE THREAD

**Issue by Sloox** [Flank/flank#2090]
**Describe the bug**
> A clear and concise description of what the bug is.
There are libraries that still rely upon bintray which will completely be offline in February 1st, 2022.
https://www.jfrog.com/confluence/display/JFROG/JFrog+Bintray+Migration+Guide

For example an error occured when bintray went offline resulting in the following error:
```
Could not resolve all dependencies for configuration ':test_runner:runtimeClasspath'.
   > Could not resolve com.google.http-client:google-http-client-appengine:1.39.2.
     Required by:
         project :test_runner > com.google.cloud:google-cloud-storage:1.117.1
      > Could not resolve com.google.http-client:google-http-client-appengine:1.39.2.
         > Could not get resource 'https://dl.bintray.com/kotlin/ktor/com/google/http-client/google-http-client-appengine/1.39.2/google-http-client-appengine-1.39.2.pom'.
            > Could not HEAD 'https://dl.bintray.com/kotlin/ktor/com/google/http-client/google-http-client-appengine/1.39.2/google-http-client-appengine-1.39.2.pom'. Received status code 502 from server: Bad Gateway
```
It resulted in large compile times and an eventual failure.
Running with --offline is Solves the problem but it is not a final solution.

**Comment by Sloox** [comment]
most likely fixed in: https://github.com/Flank/flank/pull/2091

**Comment by axelzuziak-gogo** [comment]
To be verified by @Sloox

**Comment by jan-gogo** [comment]
The problems with bintray made the repository unable to build. The problem doesn't occur right now so I am closing this issue. Is fixed by #2091
